modules/vaos/spec/services/preferences_service_spec.rb

```diff
@@ -20,4 +22,36 @@
       end
     end
   end
+
+  describe '#put_preferences' do
+    let(:user) { build(:user, :vaos) }
+    let(:request_body) do
+      {
+        notification_frequency: 'Each new message',
+        email_allowed: true,
+        email_address: 'abraham.lincoln@va.gov',
+        text_msg_allowed: false,
+        text_msg_ph_number: ''
+      }
+    end
+
+    context 'with valid params' do
+      it 'updates preferences', :skip_mvi do
```

Comment:
Why are we skipping MVI? It's not obvious to someone who's not involved w/ VAOS details, could you add a succinct comment?

Reply:
We do this for all of our more recent specs, specifically the ones that use the vaos user factory. Without this, when the user is reloaded, the instance variable that we've set in the factory somehow gets reset and mvi and ends up being re-generated from a different factory. 

The whole stub_mvi convenience method is really rather confusing and ought to be refactored or removed in my opinion. What we're doing in vaos_users.rb factory is I think more efficient.

Reply:
I would have to add this comment in A LOT of places.

Reply:
Ok - thanks for the explanation. After looking through the code it's more clear, definitely ripe for refactor. 

Also `:skip_mvi` is indicative enough.

Reply:
I haven't had a chance to go in and re-record VCR cassettes for the specs that are still using :mhv user. Most of the ones from the last month have been using :vaos user. Go ahead and try it out int he console for yourself -- you'll notice that for unit level specs things work just fine. Then for integration level specs, for whatever odd reason, stub_mvi starts doing some really wonky and unexpected things.